### ISSUE TITLE
Last week for monthlydowtrigger

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#294]
I'm having an issue creating a task for last week of the selected months.

I searched the discussions and found a thread back in 2009 with a similar issue, except it dealt with FifthWeek.

new MonthlyDOWTrigger is expecting 3 parameters.

If I pass WhichWeek.LastWeek - I get an exception saying only one single week is allowed.

If I pass some other week and then set RunOnLastWeekOfMonth = true - the task gets created for the last week and other week that has been passed when object was initialized.

Basically it's not allowing me to set last week only.

Please help.

Originally posted: 2010-10-26T12:54:44

**Comment by dahall** [comment]
Thanks for finding this bug. I have corrected it in source code change set 58843 and have updated the assembly available in the download forRelease 1.5 Beta 3.



Originally posted: 2010-10-26T14:12:10

**Comment by dahall** [comment]
Thanks dahall,



Just to clarify, if I want to create a task that will run only on last week, do I pass WhichWeek.LastWeek to the MonthlyDOWTrigger?



Or do I pass null or something else for WhichWeek and set RunOnLastWeekOfMonth = true?



Thanks again!



Originally posted: 2010-10-27T07:13:57

**Comment by dahall** [comment]
Well, I tried to just use WhichWeek.LastWeek when I initialize the trigger object and it worked, so I'm just going to omit using RunOnLastWeekOfMonth parameter.



Thanks for taking care of a new compile this quick dahall!



Originally posted: 2010-10-27T12:05:47
